agent/coordinate_endpoint.go

```diff
@@ -168,6 +168,7 @@ func (s *HTTPServer) CoordinateUpdate(resp http.ResponseWriter, req *http.Reques
 		return nil, nil
 	}
 	s.parseDC(req, &args.Datacenter)
+	s.parseToken(req, &args.Token)
```

Comment:
this looks good but don't you also need it right before the other RPCS in this method?

Reply:
what other RPC calls are there?